akka-docs/src/main/paradox/stream/stream-cookbook.md

```diff
@@ -18,7 +18,7 @@ If you need a quick reference of the available processing stages used in the rec
 In this collection we show simple recipes that involve linear flows. The recipes in this section are rather
 general, more targeted recipes are available as separate sections (@ref:[Buffers and working with rate](stream-rate.md), @ref:[Working with streaming IO](stream-io.md)).
 
-### Logging elements of a stream
+### Logging in stream
```

Comment:
Renaming the section is safe - confirmed no other place referred to this by `@ref:[***](stream-cookbook.md#loggign-elements-of-a-stream)`.

![2018-02-11_15h21_44](https://user-images.githubusercontent.com/7414320/36070514-74d2a8be-0f3f-11e8-99e9-75d1d9136a7a.png)

Reply:
New name sounds a bit weird though, perhaps making it plural would make it better: `Logging in streams`

Reply:
Thanks for the feedback. Added a new commit to fix this.